### PR TITLE
Weakpoints & Proficiencies for Zombie-Adjacent Monsters

### PR DESCRIPTION
**Pull request by SolventMercury**
#### Summary
Content "Added Proficiencies & Weakpoints to Zombie-Adjacent Monsters"

#### Purpose of change
Builds on my last pull request, #64194. Essentially the same treatment, but for the "zombie-adjacent" monsters this time. This consists of the feral humans, fungal zombies, and enemies in the jabberwock.json file.

#### Describe the solution
Added appropriate data so that enemies had weakpoints and proficiencies defined for them as needed. Also made a few minor description tweaks.

#### Describe alternatives you've considered
N/A

#### Testing
Loaded into world with the new JSONs swapped in, made sure everything worked, tested a few enemies to ensure that changes stuck.

### CODE REVIEW

File: data/json/monsters/feral_humans.json [hunk 1]
```diff
@@ -737,6 +737,7 @@
     "melee_dice_sides": 3,
     "dodge": 1,
     "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_syn_armored" ],
```
Comment (hexagonrecursion) [inline data/json/monsters/feral_humans.json:740]: Description says "body-armor **wasn't** one of their priorities". Why prof_wp_syn_armored?

File: data/json/monsters/jabberwock.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "melee_dice": 2,
     "melee_dice_sides": 4,
     "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "families": [ "prof_intro_biology", "prof_wp_demihuman" ],
```
Comment (hexagonrecursion) [inline data/json/monsters/jabberwock.json:22]: This is described as "an amalgamation". Maybe prof_wp_amalgamation?

File: data/json/monsters/jabberwock.json [hunk 3]
```diff
@@ -81,6 +83,7 @@
     "melee_dice": 4,
     "melee_dice_sides": 8,
     "melee_damage": [ { "damage_type": "cut", "amount": 3 } ],
+    "families": [ "prof_intro_biology", "prof_wp_demihuman" ],
```
Comment (hexagonrecursion) [inline data/json/monsters/jabberwock.json:86]: prof_wp_amalgamation?

**Review by hexagonrecursion — CHANGES_REQUESTED** [review]
Here are a couple nitpicks from me

File: data/json/monsters/feral_humans.json (hunk 1 above)
Comment (SolventMercury) [reply data/json/monsters/feral_humans.json:740]: Ah you're right, I misparsed that lol.

File: data/json/monsters/jabberwock.json (hunk 2 above)
Comment (SolventMercury) [reply data/json/monsters/jabberwock.json:22]: No. Amalgamations as referred to with weakpoints are a very specific *thing*, like a zombie, as opposed to being a general category, like a humanoid. Specifically, they're what happens when creatures too small to zombify have their corpses shaped by the blob into weird and alien forms. All these monsters loooong predate the addition of amalgamations.

File: data/json/monsters/feral_humans.json (hunk 1 above)
Comment (SolventMercury) [reply data/json/monsters/feral_humans.json]: Fixed

**Review by hexagonrecursion — APPROVED** [review]

**Pull request merged**